code/modules/surgery/organ_manipulation.dm

```diff
@@ -51,6 +51,28 @@
 		/datum/surgery_step/mechanic_close,
 	)
 
+/datum/surgery/organ_manipulation/mechanic/next_step(mob/living/user, modifiers)
+	if(location != user.zone_selected)
+		return FALSE
+	if(user.combat_mode)
+		return FALSE
+	if(step_in_progress)
+		return TRUE
+
+	var/try_to_fail = FALSE
+	if(LAZYACCESS(modifiers, RIGHT_CLICK))
+		try_to_fail = TRUE
+
+	var/datum/surgery_step/step = get_surgery_step()
+	if(step)
```

Comment:
early return here, like

```suggestion
	if(isnull(step))
		return FALSE
```

just for indentation purposes, realistically doesn't matter and you can ignore this but it would be a bit prettier to do it like this